test/controllers/items_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class ItemsControllerTest < ActionController::TestCase
+  setup do
+    @item = items(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:items)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create item" do
+    assert_difference('Item.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.